Makefile

```
@@ -878,15 +802,29 @@ _builtin_build_artifacts:
 # by `make fix APPLY=Y` and formatting by `make fmt APPLY=Y`, both run BEFORE
 # check. APPLY here made the same tools run twice with conflicting intents,
 # so it is rejected instead of silently honoured; FIX=1 became the `fix` verb.
-# CI=Y runs make.ci.check_gates (RULING 2: rules not skip-list).
+# Under CI=Y the run is narrowed to make.ci.check_gates --
+# the strict complement of make.ci.local_check_gates, derived at the config
+# owner so the two contexts can never overlap nor leave a gate unowned.
 _builtin_check_all: _builtin_require_environment
```

checkmake (0.3.2)

[warning] Target body for "_builtin_check_all" exceeds allowed length of 5 lines (30).

(maxbodylength)